frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segment_list_item.tsx

```diff
@@ -466,7 +465,11 @@ function _SegmentListItem({
       }}
     >
       <Dropdown
-        menu={createSegmentContextMenu()} // The overlay is generated lazily. By default, this would make the overlay
+        menu={
+          selectedSegmentIds?.includes(segment.id) && selectedSegmentIds.length > 1
```

Comment:
I'd swap the two terms:

```suggestion
          selectedSegmentIds.length > 1 && selectedSegmentIds?.includes(segment.id)
```

it will be a bit more performant, since the greater-check is faster (and if it fails, the second doesn't need to be done).

Reply:
I understand! This leads to an error though because `selectedSegmentIds` may be null and if the `includes` check is done first and the ids are null, the second one isn't done (if I understand correctly)

Reply:
Ah, my bad, then how about `(selectedSegmentIds || []).length > 1 && selectedSegmentIds?.includes(segment.id)` ?

Reply:
sounds good, I will do that!